presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -51,7 +51,9 @@ The above properties are described below:
   unique for every node. This identifier should remain consistent across
   reboots or upgrades of Presto. If running multiple installations of
   Presto on a single machine (i.e. multiple nodes on the same machine),
-  each installation must have a unique identifier.
+  each installation must have a unique identifier. The ID value can only use
```

Comment:
We say identifier multiple places, so I'd rather not introduce the shortened version for the new sentence. I think we can we just say "The value" here.

Let's change the wording a bit and remove the regex

```
The value must start with an alphanumeric character and
must only contain alphanumeric, ``-``, or ``_`` characters.

Reply:
Updated